src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -182,6 +182,33 @@ public function iSpecifyTheBillingAddressAs(AddressInterface $address): void
         $this->fillAddress('billingAddress', $address);
     }
 
+    /**
+     * @When /^the visitor try to specify the incorrect billing address as "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)" for "([^"]+)"$/
+     */
+    public function iTryToSpecifyTheIncorrectBillingAddressAs(
+        string $city,
+        string $street,
+        string $postcode,
+        string $countryName,
+        string $customerName
+    ): void {
+        $addressType = 'billingAddress';
+
+        $this->addAddress($addressType, $city, $street, $postcode, $customerName, $countryName);
+    }
+
+    /**
+     * @When /^the visitor try to specify the billing address without country as "([^"]+)", "([^"]+)", "([^"]+)" for "([^"]+)"$/
```

Comment:
You can name the arguments and then merge this method with the one above
```suggestion
     * @When the visitor try to specify the billing address without country as :city, :street, :postcode for :customerName
```